src/store/index.js

```diff
@@ -30,6 +30,8 @@ export default new Vuex.Store({
     chainNames: [],
     wizardCurrentStep: 0,
     wizardIsCollapsed: false,
+    hideMobileNavigation: false,
```

Comment:
do we need to persist the state of the navigation?

Reply:
It is implemented like so - if we have to collapse the mobile menu from a different pages and components. In this case we have the search.